src/main/java/de/hysky/skyblocker/skyblock/garden/FarmingHud.java

```diff
@@ -34,7 +34,7 @@ public class FarmingHud {
     private static final Logger LOGGER = LoggerFactory.getLogger(FarmingHud.class);
     public static final NumberFormat NUMBER_FORMAT = NumberFormat.getInstance(Locale.US);
     private static final Pattern COUNTER = Pattern.compile("Counter: (?<count>[\\d,]+) .+");
-    private static final Pattern FARMING_XP = Pattern.compile("§3\\+(?<xp>\\d+.?\\d*) Farming \\((?<percent>\\d+.?\\d*)%\\)");
+    private static final Pattern FARMING_XP = Pattern.compile("§3\\+(?<xp>\\d+.?\\d*) Farming \\((?<percent>\\d{1,3}(,\\d{3})*.?\\d*)%\\)");
```

Comment:
```suggestion
    private static final Pattern FARMING_XP = Pattern.compile("§3\\+(?<xp>\\d+.?\\d*) Farming \\((?<percent>[\\d,]+.?\\d*)%\\)");
```
<img width="692" alt="Screenshot 2024-05-04 at 10 46 47" src="https://github.com/SkyblockerMod/Skyblocker/assets/92656833/b8096b23-d521-4bf7-8e53-7fc5949a4675">

Reply:
You need to remove one of the slashes in all the double slashes